### DISCUSSION TITLE
NOT WORK / НЕ РАБОТАЕТ

### DISCUSSION THREAD

**Discussion by Zknos777** [hufrea/byedpi#174] [General]
Hi guys, sorry for my english
This settings string "sudo ciadpi --split 1 --disoob 3 --disorder 7 --fake -1 --auto none --tlsrec 3+h --mod-http=h,d,r" doesn't work with Beeline (Moscow).
Command line empty after this string and attemps load any video from youtube... Any advice?
Full source redownloaded, and remaked.

**Comment by Kurdakov** [comment]
Up to Friday some Moscow providers worked, now do not (tried various parameters). Seems they keep improving hardware capabilities.

**Comment by Zknos777** [comment]
yep. bydpi now can be unistalled

**Reply by slesru** [reply]
It was always possible to uninstall it.

Situation is different- unfortunately, it became useless since 10.10.24.
Yes, there are reports that it works, but not for me.
Roscomnadzor finally blocked anus ;-)
